src/adapters/cox.js

```diff
@@ -0,0 +1,255 @@
+                break;
+              }
+            }
+          } return { url: t, isTop: e, depth: i };
+        },
+
+        srTestCapabilities: function srTestCapabilities() {
+          var plugins = navigator.plugins,
+            flashVer = -1,
+            sf = 'Shockwave Flash';
+
+          if (plugins && plugins.length > 0) {
+            if (plugins[sf + ' 2.0'] || plugins[sf]) {
+              var swVer2 = plugins[sf + ' 2.0'] ? ' 2.0' : '';
+              var flashDescription = plugins[sf + swVer2].description;
+              flashVer = flashDescription.split(' ')[2].split('.')[0];
+            }
+          }
+          if (flashVer > 4) return 15; else return 7;
+        }
+
+      };
+    }());
+
+    // Ad calling functionality
+    CMT.Service = (function () {
+      // Closure variables shared by the service functions
+      var U = CMT.Util;
+
+      return {
+
+        buildSrc: function buildSrc(tag, env) {
+          var src = (document.location.protocol === 'https:' ? 'https://' : 'http://') + (!env || env === 'PRD' ? '' : env === 'PPE' ? 'ppe-' : env === 'STG' ? 'staging-' : '') + 'ad.afy11.net/ad' + '?mode=11' + '&ct=' + U.srTestCapabilities() + '&nif=0' + '&sf=0' + '&sfd=0' + '&ynw=0' + '&rand=' + U.getRand() + '&hb=1' + '&rk1=' + U.getRand() + '&rk2=' + new Date().valueOf() / 1000;
+
+          // Make sure we don't have a response object...
+          delete tag.response;
+
+          // Extracted url info...
+          var urlInfo = U.extractUrlInfo();
+          tag.pageUrl = urlInfo.url;
+          tag.puTop = urlInfo.isTop;
+
+          // Attach the serialized tag to our string
+          src += '&ab=' + U.encodeUriObject(tag);
+
+          return src;
+        },
+
+        getAd: function (zoneKey) {
+          if (!zoneKey) return;
+
+          return this._getData(zoneKey, 'ad') + (this._getResponse().tpCookieSync || ''); // ...also append cookie sync if present
+        },
+
+        // getSecondPrice: function getSecondPrice(zoneKey) {
+        //  if (zoneKey.substring(0, 2) !== 'as') zoneKey = 'as' + zoneKey;
+        //  var bid = this.getBidTrue(zoneKey),
+        //    floor = this._getData(zoneKey, 'floor');
+
+        //  // If no floor, just set it to 80% of the bid
+        //  if (!floor) floor = bid * 0.80;
+
+        //  // Adjust the floor if it's too high...it needs to always be lower
+        //  if (floor >= bid) {
+        //    floor = floor * 0.80; // Take off 20% to account for possible non-adjusted 2nd highest bid
+
+        //    // If it's still too high, just take 80% to 90% of the bid
+        //    if (floor >= bid) floor = bid * ((Math.random() * 10) + 80) / 100;
+        //  }
+        //  return Math.round(floor * 100) / 100;
+        // },
+
+        // setAuctionPrice: function setAuctionPrice(ad, bid) {
+        //  return ad ? ad.replace('${AUCTION_PRICE}', bid) : ad;
+        // },

```

Comment:
We have a partner that utilizes the commented-out code.  We'd rather keep it in there for completeness' sake.  But if y'all would prefer it be removed, we'll be happy to do so.  Please let me know.  Thanks!